### PR TITLE
[HAR-130] Add page component

### PR DESCRIPTION
**Pull request by michaelmaat**

### CODE REVIEW

File: docs/layouts/page.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+```
+page
+├── page__header
+├── page__main (modifier)
+├── page__footer
+```
+
+## Example HTML
+
+Basic example without inner and columns
```
Comment (edgargressie) [inline docs/layouts/page.md]: I think this description is about the layout layout

**Comment by bartvandebiezen** [comment]
We could also add the typography starting point to 'page' block? Or should this be added to the reset?

File: docs/layouts/page.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Layout
```
Comment (edgargressie) [inline docs/layouts/page.md]: The title of this document should be 'Page'

**Comment by stevengunneweg** [comment]
@bartvandebiezen how does typography relate to the page component?

**Comment by edgargressie** [comment]
@bartvandebiezen at the moment we've a default in the [body reset](https://github.com/AanZee/harbour/blob/master/scss/reset/body.scss).

**Comment by bartvandebiezen** [comment]
@edgargressie 👍

**Comment by bartvandebiezen** [comment]
In a project I used table for IE-10-11 to fix a render bug:

````
.page {
	@include media-ie-10-11 {
		display: table;
		width: 100%;
		height: 100%;
	}
}

.page__header {
	@include media-ie-10-11 {
		display: table-row;
		height: 1px;
	}
}

.page__main {
	@include media-ie-10-11 {
		padding-bottom: $spacing * 0.5;
		height: 100%;
	}
}

.page__footer {
	@include media-ie-10-11 {
		display: table-row;
		height: 1px;
	}
}
````

**Pull request merged**